app/screens/channel_info/channel_info.js

```diff
@@ -207,8 +210,20 @@ export default class ChannelInfo extends PureComponent {
                     if (result.error.server_error_id === 'api.channel.delete_channel.deleted.app_error') {
                         this.props.actions.getChannel(channel.id);
                     }
+                } else if (this.props.viewArchivedChannels) {
+                    this.props.actions.handleSelectChannel(channel.id);
+                    if (Platform.OS === 'android') {
```

Comment:
maybe add a conditional parameter to so it does not emit the event and then replace this block of code with `this.close(false)`

Reply:
:+1: